src/test/java/com/uid2/admin/vertx/test/ServiceTestBase.java

```diff
@@ -262,10 +286,17 @@ protected void setAdminUsers(AdminUser... adminUsers) {
 
     protected static Handler<AsyncResult<HttpResponse<Buffer>>> expectHttpError(VertxTestContext testContext, int errorCode) {
         return ar -> {
-            assertTrue(ar.succeeded());
-            HttpResponse response = ar.result();
-            assertEquals(errorCode, response.statusCode());
-            testContext.completeNow();
+            try {
+                HttpResponse response = ar.result();
+                assertAll(
+                        "expectHttpError",
+                        () -> assertTrue(ar.succeeded()),
+                        () -> assertEquals(errorCode, response.statusCode())
```

Comment:
I think this method is becoming very similar to methods like succceeding() or succeedingThenComplete(), possible to re-use?
https://github.com/eclipse-vertx/vertx-junit5/blob/6205224f5168042f99fff1bb5072828c0f21e69a/src/main/java/io/vertx/junit5/VertxTestContext.java#L215

Reply:
Good point, this is a good one to use succeeding instead, I've changed it.